ui/index.html

```diff
@@ -27,19 +27,19 @@
     <!-- Data tables css -->
     <link href="../node_modules/datatables.net-dt/css/jquery.dataTables.css" rel="stylesheet" type="text/css">
 
-    <!-- Jquery ui date picker css -->
-    <link href="css/jquery.datetimepicker.min.css" rel="stylesheet" type="text/css">
+	<!-- Jquery ui date picker css -->
```

Comment:
Ah no no no no. Please don't add tabs. Undo the indentation changes in this file.

In rotkehlchen we only use 4 spaces for indentation.

Reply:
Ok that is weird it should not happen. I have the editor configured to use spaces. I will have a look.